cmd/subnetcmd/configure.go

```diff
@@ -17,9 +16,11 @@ import (
 // avalanche subnet configure
 func newConfigureCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:          "configure [subnetName]",
-		Short:        "",
-		Long:         ``,
+		Use:   "configure [subnetName]",
+		Short: "Allows to add additional config files for the avalanchego nodes",
+		Long: `Avalanchego nodes can be configured at different levels. 
+For example, subnets have their own subnet config (applies to all chains/VMs in the subnet).
+And each chain or VM can have its own specific chain config file.`,
```

Comment:
Need to add that this command allows you to set both the subnet config and the chain config